test/data/build-rpms-and-repos.sh

```diff
@@ -30,10 +30,12 @@ function build_rpms() {
     # use a temporary topdir to avoid writing to user's rpmbuild directory
     RPMBUILD_TOPDIR=$(mktemp -d)
 
+    # Note: _build_name_fmt requires escaped %% for use in headerSprintf()
     rpmbuild -ba \
         --define="_topdir ${RPMBUILD_TOPDIR}" \
-        --define="_srcrpmdir ${TARGET_DIR}/src" \
+        --define="_srcrpmdir ${TARGET_DIR}" \
         --define="_rpmdir ${TARGET_DIR}" \
+        --define="_build_name_fmt %%{NAME}-%%{VERSION}-%%{RELEASE}.%%{ARCH}.rpm" \
```

Comment:
The reason for this isn't obvious to me, so might warrant an explanatory comment...

Tbh. this commit should get some description about the reasoning behind it (the root cause being Copr and what's going on with it). Or maybe in the code comment, so that one doesn't have to dig in history...